drivers/google/compute_util.go

```diff
@@ -292,6 +415,29 @@ func (c *ComputeUtil) createInstance(d *Driver) error {
 		},
 	}
 
+	// This is a workaround to a known issue in the GCE API which prevents the standard .List() function from filtering
+	// instances based off of network tags https://issuetracker.google.com/issues/143463446#comment9,
+	// instead we use a label which equals the name of the generated firewall rule.
+	if len(d.OpenPorts) > 0 {
+		if instance.Labels != nil {
```

Comment:
FWIW I don't think `instance.Labels` can be `nil` here, but if we really want to add a check if put it line 418.

Reply:
You're right, I added these checks before refactoring `parseLabels`, the field should always be non-nil now